src/test/java/com/google/api/codegen/testdata/nodejs/nodejs_library.baseline

```diff
@@ -123,38 +123,39 @@ $ npm install --save @google-cloud/library
  */
 'use strict';
 
-describe('LibraryServiceSmokeTest', function() {
+describe('LibraryServiceSmokeTest', () => {
   if (!process.env.SMOKE_TEST_PROJECT) {
     throw new Error("Usage: SMOKE_TEST_PROJECT=<project_id> node #{$0}");
   }
   var projectId = process.env.SMOKE_TEST_PROJECT;
 
-  it('successfully makes a call to the service', function(done) {
-    var library = require('../src');
+  it('successfully makes a call to the service', done => {
+    const library = require('../src');
 
-    var client = library.v1({
+    var client = new library.v1.LibraryServiceClient({
       // optional auth parameters.
     });
 
     var formattedName = client.bookPath("testShelf-" + Date.now().toString(), projectId);
     var rating = library.v1.types.Book.Rating.GOOD;
```

Comment:
@landrito This line is now wrong, since we nixed `types`. It should just be the string `'GOOD'`. Any idea how to change that? (It was the only thing I could not track down.)

Reply:
I think if you changed this line:
https://github.com/googleapis/toolkit/blob/9402fbbe3c393f121f528ece2003b7a418188218/src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSModelTypeNameConverter.java#L187

to:
```java
return TypedValue.create(getTypeName(type), String.format("'%s'", value.getSimpleName()));

You'll get what you're looking for.

Reply:
What do you mean we nixed `types`? Will the user still have access to them?